src/providers.ts

```diff
@@ -55,6 +55,7 @@ const signIn = async (connector: Providers) => {
     ssx = new SSX({
         providers: {
             web3: { driver: provider },
+            server: { host: "/" },
```

Comment:
This enables `ssx-server` communication at this url. Because the server is on the same host, we just use `/`.